torchvision/csrc/io/decoder/gpu/gpu_decoder.cpp

```diff
@@ -29,8 +29,7 @@ torch::Tensor GPUDecoder::decode() {
   unsigned long videoBytes = 0;
   uint8_t* video = nullptr;
   at::cuda::CUDAGuard device_guard(device);
-  auto options = torch::TensorOptions().dtype(torch::kU8).device(torch::kCUDA);
-  torch::Tensor frame = torch::zeros({0}, options);
```

Comment:
This is a common idiom across ops:
https://github.com/pytorch/vision/blob/bae1d7e2c73a456b52f0ccafea411c9c4fbf2ef8/torchvision/csrc/ops/cuda/nms_kernel.cu#L106-L109

Reply:
This was suggested by Francisco. Check https://github.com/pytorch/vision/pull/5019#discussion_r776807411.

Reply:
Oh I see. I missed it as it was not part of the modifications.